src/collections/componentSet.ts

```diff
@@ -711,7 +711,7 @@ export class ComponentSet extends LazyCollection<MetadataComponent> {
       try {
         version = `${await getCurrentApiVersion()}.0`;
       } catch (e) {
-        version = '58.0';
```

Comment:
I only bumped this because I was getting this error (10 times) when running `yarn test`:

```
  625 passing (23s)
  3 pending
  10 failing

  1) Labels
       verify md files:

      AssertionError [ERR_ASSERTION]: test/snapshot/sampleProjects/bots/__snapshots__/verify-md-files.actual/package.xml
      + expected - actual

           <types>
               <members>TestSkill</members>
               <name>Skill</name>
           </types>
      -    <version>58.0</version>
      +    <version>60.0</version>
       </Package>
      
      at snap (node_modules/mocha-snap/dist/snap.js:82:31)
      at async Context.<anonymous> (test/snapshot/sampleProjects/bots/snapshots.test.ts:35:7)
```

Reply:
I'm actually going to revert this change. I only did this because I was getting an error when running `yarn test`. One headache later... it turns out the security software IT installed on my work machine was blocking the request to app-exchange for the latest API version. 🙃